openpype/hosts/maya/plugins/publish/validate_path_attributes.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from maya import cmds
+
+import pyblish.api
+
+from openpype.pipeline.publish import ValidateContentsOrder
+
+
+class ValidatePathForPlugin(pyblish.api.InstancePlugin):
+    """
+    Ensure Paths in Non-Maya Nodes(from plugins
+    such as Yeti, AbcExport) are correct
+    """
```

Comment:
```suggestion
    """Validate plug-in path attributes point to existing file paths."""
```

Also `AbcExport`? I think you meant `AbcImport`? Anyway, My suggestion simplifies that away anyway.